WalletWasabi/Crypto/StringCipher.cs

```diff
@@ -1,114 +1,115 @@
-using System.IO;
 using System.Linq;
 using System.Security.Cryptography;
 using System.Text;
 using WalletWasabi.Crypto.Randomness;
 
 namespace WalletWasabi.Crypto
 {
-	// https://stackoverflow.com/a/10177020/2061103
 	public static class StringCipher
 	{
 		// This constant is used to determine the keysize of the encryption algorithm in bits.
 		// We divide this by 8 within the code below to get the equivalent number of bytes.
-		private const int KeySize = 128;
+		private const int KeyByteSize = 128 / 8;
+		private const int AesBlockByteSize = 128 / 8;
+		private const int PasswordSaltByteSize = 128 / 8;
+		private const int SignatureByteSize = 256 / 8;
 
 		// This constant determines the number of iterations for the password bytes generation function.
 		private const int DerivationIterations = 1000;
```

Comment:
The number of iterations is small for current hardware.

Some references:

* https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html#pbkdf2
* https://github.com/zkSNACKs/WalletWasabi/pull/6946/files#diff-3515565776ab9676fffaa3331c1de21ebcd30d829334670c055b66e146f83ac1R14
* https://github.com/zkSNACKs/WalletWasabi/pull/6936#issuecomment-1000666818